Gemfile

```diff
@@ -44,7 +44,7 @@ group :development do
 end
 
 group :test do
-  gem 'refinerycms-testing', path: "./testing"
+  gem 'refinerycms-testing'
```

Comment:
i think we should keep the local path it's easier to update the project in this context

Reply:
Did you see my PR @anitagraham ? https://github.com/anitagraham/refinerycms/pull/4